modules/Hoot/ui/managePanel/jobs.js

```diff
@@ -173,13 +173,26 @@ export default class Jobs extends Tab {
         }
     }
 
+    jobsHistoryTotalCheck(jobsHistory) {
+        let activeTotal = this.total;
+
+        if ( activeTotal && activeTotal < jobsHistory.total ) {
+            return true;
+        }
+    }
+
     async loadJobs() {
         let jobsRunning = await Hoot.api.getJobsRunning();
         let jobsHistory = await Hoot.api.getJobsHistory(this.params);
-        await Hoot.layers.refreshLayers();
+
+        let checkTotal = this.jobsHistoryTotalCheck(jobsHistory);
+
+        if ( jobsHistory.total > 0 && checkTotal ) {
+            await Hoot.layers.refreshLayers();
+        }
```

Comment:
Instead of all this, can we just do this?
`if ( jobsHistory.total > this.total ) {`